apps/extension/package.json

```diff
@@ -20,7 +20,7 @@
     "@bufbuild/protobuf": "^1.9.0",
     "@connectrpc/connect": "^1.4.0",
     "@connectrpc/connect-web": "^1.4.0",
-    "@penumbra-labs/registry": "^5.1.0",
+    "@penumbra-labs/registry": "npm:@penumbra-zone/test-publish@0.7.0",
```

Comment:
To be swapped with `7.0.0` when published

Reply:
can we simply use `latest`

Reply:
It's not a bad idea, but given we are on v7, we've had 7 breaking changes of the registry api to date. Think this is worth revisiting when that stabilizes a bit. Otherwise, we'd risk dev env breaking on a registry update.